pkg/app/schema.go

```diff
@@ -171,25 +169,18 @@ func write(fs afero.Fs, appRoot string, spec *Spec) error {
 		return errors.Wrap(err, "write app.yaml")
 	}
 
-	if err = cleanOverride(fs, appRoot); err != nil {
+	if err = removeOverride(fs, appRoot); err != nil {
```

Comment:
I have questions around transactionality - if `SaveOverride` fails below, is it ok that we overwrote the `app.yaml` above?

Reply:
That's an issue which should be tracked. I'm creating an issue for it.